framework/ldtest/filter.go

```diff
@@ -121,3 +121,26 @@ func PrintFilterDescription(filters RegexFilters, allCapabilities []string, supp
 		}
 	}
 }
+
+func BriefFilterDescription(filters RegexFilters) string {
+	var parts []string
+	if filters.MustMatch.IsDefined() {
+		parts = append(parts, fmt.Sprintf("must match %s", filters.MustMatch))
+	}
+	if filters.MustNotMatch.IsDefined() {
+		parts = append(parts, fmt.Sprintf("must not match %s", filters.MustNotMatch))
+	}
+	if len(parts) == 0 {
+		return "none"
+	}
+	return strings.Join(parts, ", ")
+}
+
+func autoEscapeTestRegex(pattern string) string {
```

Comment:
I added this logic because Matthew very reasonably tried to use a filter expression like `-run 'evaluation/param/general (secondary)'` and got unexpected results due to the regex behavior of parens.